client/client.go

```diff
@@ -201,6 +201,9 @@ type (
 
 		// ListClosedWorkflow gets closed workflow executions based on request filters.
 		// Retrieved workflow executions are sorted by start time in descending order.
+		// (Retrieved workflow executions could also be sorted by closed time in descending order,
+		// if cadence server side config EnableReadFromClosedExecutionV2 is set to true.)
```

Comment:
Shouldn't this flag be set to true by default?

Reply:
I personally also like to default true. But for backward compatibility I leave it default false.